harvester/products/sources/anatomy_tool.py

```diff
@@ -0,0 +1,240 @@
+        return {
+            "root": root,
+            "departments": [],
+            "associates": []
+        }
+
+    @classmethod
+    def get_publishers(cls, soup: bs4.BeautifulSoup, el: bs4.element.Tag) -> list[str]:
+        return ["AnatomyTOOL"]
+
+    @classmethod
+    def get_consortium(cls, soup: bs4.BeautifulSoup, el: bs4.element.Tag) -> str:
+        return "AnatomyTOOL"
+
+    @classmethod
+    def get_publisher_date(cls, soup: bs4.BeautifulSoup, el: bs4.element.Tag) -> None | str:
+        publisher = el.find(string='Created')
+        if not publisher:
+            return
+        contribution = publisher.find_parent('contribute')
+        if not contribution:
+            return
+        datetime = contribution.find('datetime')
+        if not datetime:
+            return
+        datetime = date_parser(datetime.text)
+        return datetime.strftime("%Y-%m-%d")
+
+    @classmethod
+    def get_publisher_year(cls, soup: bs4.BeautifulSoup, el: bs4.element.Tag) -> None | int:
+        publisher_date = cls.get_publisher_date(soup, el)
+        if publisher_date is None:
+            return
+        datetime = date_parser(publisher_date)
+        return datetime.year
+
+    @classmethod
+    def get_lom_educational_levels(cls, soup: bs4.BeautifulSoup, el: bs4.element.Tag) -> list[str]:
+        return ["HBO", "WO"]
+
+    @classmethod
+    def get_copyright_description(cls, soup: bs4.BeautifulSoup, el: bs4.element.Tag) -> None | str:
+        node = el.find('rights')
+        if not node:
+            return
+        description = node.find('description')
+        return description.find('string').text.strip() if description else None
+
+    @classmethod
+    def get_learning_material_disciplines(cls, soup: bs4.BeautifulSoup, el: bs4.element.Tag) -> list[str]:
+        return ["gezondheid"]
+
+
+ANATOMY_TOOL_EXTRACTION_OBJECTIVE = {
+    "@": AnatomyToolExtraction.get_oaipmh_records,
+    "external_id": AnatomyToolExtraction.get_oaipmh_external_id,
+    "set": lambda soup, el: f"anatomy_tool:{AnatomyToolExtraction.get_oaipmh_external_id(soup, el)}",
+    "state": AnatomyToolExtraction.get_oaipmh_record_state,
+    "files": AnatomyToolExtraction.get_files,
+    "title": AnatomyToolExtraction.get_title,
+    "language": AnatomyToolExtraction.get_language,
+    "keywords": AnatomyToolExtraction.get_keywords,
+    "description": AnatomyToolExtraction.get_description,
+    "copyright": AnatomyToolExtraction.get_copyright,
+    "authors": AnatomyToolExtraction.get_authors,
+    "provider": AnatomyToolExtraction.get_provider,
+    "organizations": AnatomyToolExtraction.get_organizations,
+    "publishers": AnatomyToolExtraction.get_publishers,
+    "publisher_date": AnatomyToolExtraction.get_publisher_date,
+    "publisher_year": AnatomyToolExtraction.get_publisher_year,
+    "copyright_description": AnatomyToolExtraction.get_copyright_description,
+    "doi": lambda soup, el: None,
+    # Learning material metadata
```

Comment:
No studies here either. Fine by me! Should be removed eventually anyway.

Reply:
Added since the change to deal with missing studies with Edusources will be deployed this Monday. Better safe than sorry.